go.mod

```
@@ -1,5 +1,11 @@
-module github.com/sabhiram/go-gitignore
+module github.com/Diogenesoftoronto/go-gitignore
 
-go 1.13
+go 1.22
```

Maybe we can leave the minimum possible version the library is compiled with:
```suggestion
go 1.16
```